controller/lib/core/sensors.cpp

```diff
@@ -107,3 +141,15 @@ float get_pressure_reading(AnalogPinId pinId) {
   // Sensitivity of all pressure sensors is 1 V/kPa; no division needed.
   return ((float)runningSum * ADC_LSB);
 }
+
+float convert_diff_pressure_to_volumetric_flow(
+    float diffPressureInKiloPascals) {
+  if (diffPressureInKiloPascals > 0.0f) {
+    return ROOT_OF_TWO_OVER_DENSITY_OF_AIR * venturiAreaProduct *
```

Comment:
We can avoid repeating the function by doing something like:

```
float sgn = copysignf(1, diffPressureInKiloPascals)
return sgn * ... sqrtf(sgn * ...)
```

Reply:
Neat!